packages/core/src/data-import/providers/import-parser/import-parser.ts

```diff
@@ -46,13 +80,16 @@ export interface ParsedProduct {
     slug: string;
```

Comment:
Are these "main" translatable properties still needed? As far as I can see, all translations put in the `translations` array, and the mainLanguage ones are duplicated in the root object. Is my understanding correct?

Reply:
Correct, I'm adding all translations to the `translations` arrays, because that makes the most sense for subsequent importing. I added the whole `mainLanguage` duplication part only to keep `ParsedProduct` etc. compatible with the current implementation. I personally don't need these 'main' translations at all and would be happy to remove those properties along with the `mainLanguage` parameter. But I'm not sure if that would break things for anyone else...

Reply:
I think we can reasonably consider this an internal API, in the sense that it is not documented and is not exported from the `@vendure/core` package, so if anyone _did_ rely on it, they would need to do a deep import, which is in itself a bit of a signal that it is not a public API.

So I would say it is ok to remove the top-level props that are duplicated, and clean up the rest of the PR accordingly.

Reply:
Is it safe to put all custom fields, including those that are not of type `localeString`, into `translations.customFields`, and in return, to put `localeString` fields of a random translation into the "main" `customFields` property? I think this should work, but I'm not sure. Otherwise, we would have to keep at least the `customFields` "main" property.

Reply:
And on second thought, we need to keep the `mainLanguage` parameter after all, in order to be able to set the import language for untranslated CSV files to `ctx.languageCode`.

Reply:
Kept the `mainLanguage` parameter but removed those top-level translatable fields.